### PR TITLE
LPS-132545 Change MultiPanelSidebar width to 368px

### PR DESCRIPTION
**Pull request by interaminense**
ticket: https://issues.liferay.com/browse/LPS-132545

### CODE REVIEW

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/components/field-sets/FieldSetModal.scss [hunk 1]
```diff
@@ -1,15 +1,17 @@
-.fieldset-modal {
-	.data-engine-form-builder > .sheet {
-		background-color: transparent;
-		border: 0 none;
-	}
+$multiPanelSidebarContentWidth: 368px;
```
Comment (miltonmc) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/components/field-sets/FieldSetModal.scss:1]: Import `_vatiables.scss` to get access to `$multiPanelSidebarContentWidth`

```suggestion
@import '../../../css/variables';
```

**Review by miltonmc — CHANGES_REQUESTED** [review]
Just a minor comment

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/components/field-sets/FieldSetModal.scss (hunk 1 above)
Comment (interaminense) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/components/field-sets/FieldSetModal.scss:1]: Hi @miltonmc. The variable css is in another module, so I don't have access to get it

**Comment by interaminense** [comment]
ci:test:relevant

**Review by miltonmc — APPROVED** [review]

**Comment by miltonmc** [comment]
ci:forward

**Pull request closed without merging**